# This file is distributed under the same license as the Django package.
#
# Translators:
# Ali Nikneshan <ali@nikneshan.com>, 2012-2013,2020
# Alireza Savand <alireza.savand@gmail.com>, 2013
# Ali Vakilzade <ali.vakilzade@gmail.com>, 2015
# Jannis Leidel <jannis@leidel.info>, 2011
# Pouya Abbassi, 2016
# Saeed <sd.javadi@gmail.com>, 2011
msgid ""
msgstr ""
"Project-Id-Version: django\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2022-02-08 19:42+0330\n"
"PO-Revision-Date: 2020-11-06 08:17+0000\n"
"Last-Translator: Ali Nikneshan <ali@nikneshan.com>\n"
"Language-Team: Persian (http://www.transifex.com/django/django/language/"
"fa/)\n"
"Language: fa\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n > 1);\n"

#: venv/lib/python3.8/site-packages/django/contrib/gis/apps.py:9
msgid "GIS"
msgstr "جی‌آی‌اس"

#: venv/lib/python3.8/site-packages/django/contrib/gis/db/models/fields.py:64
msgid "The base GIS field."
msgstr "پایه‌ی سیستم اطلاعات جغرافیایی میدانی."

#: venv/lib/python3.8/site-packages/django/contrib/gis/db/models/fields.py:205
msgid ""
"The base Geometry field — maps to the OpenGIS Specification Geometry type."
msgstr "پایه‌ی فیلد هندسی --نوع خاص هندسه‌ی نقشه‌ها‌ی نوع OpenGIS"

#: venv/lib/python3.8/site-packages/django/contrib/gis/db/models/fields.py:289
msgid "Point"
msgstr "نقطه"

#: venv/lib/python3.8/site-packages/django/contrib/gis/db/models/fields.py:296
msgid "Line string"
msgstr "رشته خط"

#: venv/lib/python3.8/site-packages/django/contrib/gis/db/models/fields.py:303
msgid "Polygon"
msgstr "چندضلعی"

#: venv/lib/python3.8/site-packages/django/contrib/gis/db/models/fields.py:310
msgid "Multi-point"
msgstr "چند نقطه ای"

#: venv/lib/python3.8/site-packages/django/contrib/gis/db/models/fields.py:317
msgid "Multi-line string"
msgstr "چند خط رشته"

#: venv/lib/python3.8/site-packages/django/contrib/gis/db/models/fields.py:324
msgid "Multi polygon"
msgstr "چندین چند ضلعی "

#: venv/lib/python3.8/site-packages/django/contrib/gis/db/models/fields.py:331
msgid "Geometry collection"
msgstr "مجموعه هندسی"

#: venv/lib/python3.8/site-packages/django/contrib/gis/db/models/fields.py:337
msgid "Extent Aggregate Field"
msgstr "تراکم وسعت حوزه‌ی "

#: venv/lib/python3.8/site-packages/django/contrib/gis/db/models/fields.py:352
msgid "Raster Field"
msgstr "حوزه‌ی شطرنجی"

#: venv/lib/python3.8/site-packages/django/contrib/gis/forms/fields.py:20
msgid "No geometry value provided."
msgstr "مقدار جغرافیایی‌ای مقرر نشده است."

#: venv/lib/python3.8/site-packages/django/contrib/gis/forms/fields.py:21
msgid "Invalid geometry value."
msgstr "مقدار جغرافیایی نامعتبر"

#: venv/lib/python3.8/site-packages/django/contrib/gis/forms/fields.py:22
msgid "Invalid geometry type."
msgstr "نوعِ جغرافیایی نامعتبر"

#: venv/lib/python3.8/site-packages/django/contrib/gis/forms/fields.py:23
msgid ""
"An error occurred when transforming the geometry to the SRID of the geometry "
"form field."
msgstr "مشکلی در هنگام انتقال مختصات هندسی از فیلد به SRID رخ داد."

#: venv/lib/python3.8/site-packages/django/contrib/gis/templates/gis/admin/openlayers.html:26
#: venv/lib/python3.8/site-packages/django/contrib/gis/templates/gis/openlayers.html:12
msgid "Delete all Features"
msgstr "حذف همه‌ی اشیاء"

#: venv/lib/python3.8/site-packages/django/contrib/gis/templates/gis/admin/openlayers.html:28
msgid "WKT debugging window:"
msgstr "پنجره‌ی اشکال زدایی «متن قابل درک»"

#: venv/lib/python3.8/site-packages/django/contrib/gis/templates/gis/openlayers.html:13
msgid "Debugging window (serialized value)"
msgstr "پنجره‌ی اشکال زدایی (ارزش متوالی)"

#: venv/lib/python3.8/site-packages/django/contrib/gis/views.py:8
msgid "No feeds are registered."
msgstr "هیچ فیدی ثبت شده است."

#: venv/lib/python3.8/site-packages/django/contrib/gis/views.py:14
#, python-format
msgid "Slug %r isn’t registered."
msgstr "Slug %r ثبت نشده"
